Code/GraphMol/Descriptors/AtomicEnvironmentVector_catch.cpp

```diff
@@ -0,0 +1,331 @@
+//
+//  Copyright (C) 2020 Manan Goel
+//   @@ All Rights Reserved @@
+//  This file is part of the RDKit.
+//  The contents are covered by the terms of the BSD license
+//  which is included in the file license.txt, found at the root
+//  of the RDKit source tree.
+//
+
+#include "RDGeneral/test.h"
+#include "catch.hpp"
+
+#include <RDGeneral/Invariant.h>
+#include <GraphMol/RDKitBase.h>
+#include <GraphMol/FileParsers/MolSupplier.h>
+#include <GraphMol/FileParsers/FileParsers.h>
+#include <RDGeneral/RDLog.h>
+#include <vector>
+#include <algorithm>
+#include <fstream>
+#include <string>
+#include <sstream>
+#include <iterator>
+#include <GraphMol/GraphMol.h>
+#include <GraphMol/MolOps.h>
+#include <boost/tokenizer.hpp>
+#include <GraphMol/PeriodicTable.h>
+#include <GraphMol/Descriptors/AtomicEnvironmentVector.h>
+#include <Eigen/Dense>
+
+using namespace Eigen;
+
+std::vector<std::string> tokenize(const std::string &s) {
+  boost::char_separator<char> sep(", \n\r\t");
+  boost::tokenizer<boost::char_separator<char>> tok(s, sep);
+  std::vector<std::string> tokens;
+  std::copy(tok.begin(), tok.end(),
+            std::back_inserter<std::vector<std::string>>(tokens));
+  return tokens;
+}
+
+std::vector<std::vector<double>> EigenMatToSTLVector(ArrayXXd aev) {
+  std::vector<std::vector<double>> aevOutput;
+  for (auto i = 0; i < aev.rows(); i++) {
+    std::vector<double> row;
+    for (auto j = 0; j < aev.cols(); j++) {
+      row.push_back(aev(i, j));
+    }
+    aevOutput.push_back(row);
+  }
+  return aevOutput;
+}
+
+TEST_CASE("Symmetry Function Accuracy", "[Symmetry Function]") {
+  SECTION("CH4") {
+    std::string pathName = getenv("RDBASE");
+
+    std::string molFile =
+        pathName + "/Code/GraphMol/Descriptors/test_data/CH4.mol";
+
+    std::string fNameSF =
+        pathName + "/Code/GraphMol/Descriptors/test_data/CH4.out";
+
+    std::ifstream instrmSF;
+    instrmSF.open(fNameSF);
```

Comment:
It's not important, but as a general point it's more compact to just do:
```suggestion
    std::ifstream instrmSF(fNameSF);
```